apps/server/src/modules/h5p-editor/service/libraryStorage.service.ts

```diff
@@ -31,21 +34,17 @@ export class LibraryStorage implements ILibraryStorage {
 	 * @param filename the requested file
 	 */
 	private checkFilename(filename: string): void {
-		if (path.normalize(filename).startsWith(`..${path.sep}`) || path.isAbsolute(filename)) {
-			throw new Error('Illegal filename');
+		if (/\.\.\//.test(filename)) {
```

Comment:
test in separat declaration and check for variable here, makes it much more readable

Reply:
Refactored for better readability